scripts/tests/lib.sh

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
-
```

Comment:
`set -euo pipefail` was messing up my interactive Bash session. It's already at the top of every normal script, and having it in these scripts that are meant to be imported with `source` messes up interactive Bash, so I removed it.